### ISSUE TITLE
[Info] workaround for 'Fatal: Logical disk size 0'

### ISSUE THREAD

**Issue by syl1101** [Lanchon/REPIT#38]
Hello and first of all a **huge thanks** goes out **for your excellent work**! You saved my a**. :)

This is not an issue but can't find a Q&A section or a forum (sadly, i've red what's happened...) where i can post so writing here, apologising if it's not the correct place.

So, i messed with partitions on GT-I9100 using  Irwenzhao's method 1 (http://forum.xda-developers.com/galaxy-s2/orig-development/repartition-memory-s2-t3018749) which did the partitioning but unfortunately left system partition untouched at factory 512 MiB, left preload untouched and messed up something with emmc in my case. So i ended up at Repit.

Used

**lanchon-repit-20160415-system=1.0-data=same-sdcard=max-preload=min+wipe-i9100.zip**

with TWRP 3 and got the error message '**Fatal: Logical disk size 0**' for sdcard while flashing from /tmp (flashing from external SD failed with unable to unmount internal sd).

While Repit was unable to cope with the situation with the above config, simply adding +wipe for every partition solved the issue: flashing 

**lanchon-repit-20160415-system=1.0+wipe-data=same+wipe-sdcard=max+wipe-preload=min+wipe-i9100.zip**

went smoothly (from /tmp of course), Repit deleted and recreated the partitions in the correct way and now everything's as it should be (hopefully :) ).

Unfortunately i've forgot to copy the log file from /tmp in the run, all i can remember the partition was exFAT and the above given error message. I hope it's helpful in some way.

Feel free to modify or cut this post in case it's too wordy.

Again, thank you for being and writing awesome software!

**Comment by Lanchon** [comment]
hi,
thanks for the feedback.

this behavior is by design. when repit is told to KEEP a partition, its file system has to be understandable and fixable before repit will proceed. in your case, there was corruption due to some process you ran beforehand that is unknown to me.

however, when it is told to WIPE, the partition is not accessed at all before repitting, and the process will succeed even if there is corruption to begin with. this is specifically design to repair broken setups easily (such as yours) and also to allow changing file system types.

since this behavior is by design, i'll be closing this report.
but thanks for the feedback anyway!
